src/modules/user/services/ShowProfileUserService.ts

```diff
@@ -0,0 +1,26 @@
+import AppError from '@shared/infra/http/errors/AppError';
+import { inject, injectable } from 'tsyringe';
+import User from '../infra/typeorm/entities/User';
+import IUserRepository from '../repositories/IUserRepository';
+
+export interface IRequest {
+  user_id: string;
+}
+
+@injectable()
+export default class ShowProfileService {
```

Comment:
```suggestion
export default class ShowProfileUserService {
```

Thus, the name of the class is semantic with the name of the file